src/Uno.Extensions.Navigation.UI/Navigators/ControlNavigator.cs

```diff
@@ -50,9 +38,9 @@ public abstract class ControlNavigator<TControl> : ControlNavigator
         }
 
         var route = request.Route;
-        var mapping = Mappings.FindView(route);
-        Logger.LogDebugMessage($"Navigating to path '{route.Base}' with view '{mapping?.ViewType?.Name}'");
-        var executedPath = await Show(route.Base, mapping?.ViewType, route.Data);
+        var mapping = RouteResolver.Find(route);
+        Logger.LogDebugMessage($"Navigating to path '{route.Base}' with view '{mapping?.View?.Name}'");
```

Comment:
We'll need to change this to avoid interpolating message for debug messages. The only pattern that currently works (until we get the delegated interpolation) is `if(Logger.IsEnabled(Debug))`.